### PR TITLE
add deployment action in CI

### PR DESCRIPTION
**Pull request by Williangalvani**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  workflow_dispatch:
+  # NOTE: caches may be removed if not run weekly
+  #  -> may be worth scheduling for every 6 days
+
+jobs:
+  deploy-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy BlueOS Extension
+        uses: BlueOS-community/Deploy-BlueOS-Extension@v1
```
Comment (ES-Alexander) [inline .github/workflows/deploy.yml]:
```suggestion
        uses: BlueOS-community/Deploy-BlueOS-Extension@v1.0.1
```

We made a small update recently so it doesn't run as long when there aren't credentials provided - may as well use the latest :-)

**Review by ES-Alexander — APPROVED** [review]
Assuming you don't want to add the timeouts CI is complaining about, or the docker credentials, LGTM :-)

**Comment by Williangalvani** [comment]
> Assuming you don't want to add the timeouts CI is complaining about, or the docker credentials, LGTM :-)

the credentials are there, I'm confused

**Comment by ES-Alexander** [comment]
> the credentials are there, I'm confused

@Williangalvani remembered that apparently secrets are disabled for PRs, which does make some sense.

**Pull request merged**